From 5dc9594ec152362a84de341722deb50b2642f67d Mon Sep 17 00:00:00 2001
From: Andrew Goodbody <andrew.goodbody@cambrionix.com>
Date: Wed, 26 Jul 2017 13:45:49 +0100
Subject: [PATCH 2/2] Move cache dir

---
 libopkg/opkg_conf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libopkg/opkg_conf.h b/libopkg/opkg_conf.h
index 9cfcc88..b2c43e6 100644
--- a/libopkg/opkg_conf.h
+++ b/libopkg/opkg_conf.h
@@ -38,7 +38,7 @@ extern "C" {
 #define OPKG_CONF_DEFAULT_LISTS_DIR     VARDIR "/lib/opkg/lists"
 #define OPKG_CONF_DEFAULT_INFO_DIR      VARDIR "/lib/opkg/info"
 #define OPKG_CONF_DEFAULT_STATUS_FILE   VARDIR "/lib/opkg/status"
-#define OPKG_CONF_DEFAULT_CACHE_DIR     VARDIR "/cache/opkg"
+#define OPKG_CONF_DEFAULT_CACHE_DIR     VARDIR "/volatile/cache/opkg"
 #define OPKG_CONF_DEFAULT_CONF_FILE_DIR SYSCONFDIR "/opkg"
 #define OPKG_CONF_DEFAULT_LOCK_FILE     "/run/opkg.lock"
 
-- 
2.7.4

